compiler/record-minmax/driver/Driver.cpp

```diff
@@ -20,6 +20,7 @@
 #include <vconone/vconone.h>
 
 #include <luci/UserSettings.h>
+#include <luci/Log.h>
```

Comment:
This module uses _luci_ but itself is _record-minmax_ , thus I think seeing LUCI prefix may confuse the users reading log.
Defining LOG will take some time so please add a TODO
```suggestion
// TODO declare own log signature of record-minmax
#include <luci/Log.h>
```

Reply:
Done